### ISSUE TITLE
EXPOSE UDP port range fails to parse

### ISSUE THREAD

**Issue by JamesJJ** [hadolint/hadolint#192]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

`EXPOSE 1000-2000/udp`

==> pass parsing

### Actual behavior

`Dockerfile:51:19 unexpected '/' expecting digit, space or a new line followed by the next instruction`

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter 1.6.2-no-git
```

**Additional environment details (OS, stack version, etc.)**

 * Related to #38 
 * Opened upstream FR here: https://github.com/hadolint/language-docker/issues/30
